robottelo/config/base.py

```diff
@@ -395,6 +395,10 @@ def __init__(self, *args, **kwargs):
         self.external_registry_1 = None
         self.external_registry_2 = None
         self.unix_socket = None
+        self.private_registry_url = None
+        self.private_registry_name = None
```

Comment:
Same question about _name -> _image

Reply:
`docker_upstream_name=settings.docker.private_registry_name`

Reply:
If APi called it `docker_upstream_image` or `_image_name`, dont you think it will confuse matching `name->image`

Reply:
understood